### PR TITLE
Update sig-cloud-provider charter to support orphaned subprojects

### PR DESCRIPTION
**Pull request by frapposelli**
As discussed during the SIG-Cloud Provider meeting on 4/17/2019.

/cc @jagosan @hogepodge @andrewsykim

### CODE REVIEW

**Comment by andrewsykim** [comment]
cc @d-nishi

File: sig-cloud-provider/CHARTER.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ SIG Cloud Provider’s mission is to simplify, develop, and maintain cloud provi
 - core controllers (started by the cloud-controller-manager) that interact with cloud provider resources
 - all [cloud provider repositories](https://github.com/kubernetes?utf8=%E2%9C%93&q=cloud-provider-&type=&language=) under the Kubernetes organization
 - [e2e tests for cloud provider specific](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/cloud) functionality
+- all the subprojects formerly owned by [SIG-AWS](https://github.com/kubernetes/community/tree/master/sig-aws#subprojects), [SIG-AZURE](https://github.com/kubernetes/community/tree/master/sig-azure#subprojects), [SIG-GCP](https://github.com/kubernetes/community/tree/master/sig-gcp#subprojects), [SIG-IBMCloud](https://github.com/kubernetes/community/tree/master/sig-ibmcloud#subprojects), [SIG-Openstack](https://github.com/kubernetes/community/tree/master/sig-openstack#subprojects), [SIG-VMware](https://github.com/kubernetes/community/tree/master/sig-vmware#subprojects).
```
Comment (andrewsykim) [inline sig-cloud-provider/CHARTER.md]: sgtm, let's lazy consensus this timeboxed to the next SIG meeting?

Comment (frapposelli) [reply sig-cloud-provider/CHARTER.md]: SGTM

**Comment by andrewsykim** [comment]
/approve
/lgtm
/hold

Following lazy consensus for this charter update time boxed to May 1st, mailing list announcement here https://groups.google.com/forum/#!topic/kubernetes-sig-cloud-provider/3oi-bbPX96E

**Pull request merged**